### ISSUE TITLE
Add point out aliases

### ISSUE THREAD

**Issue by erikquinn** [vZMA/ZMA#14]
`.po` - point out on route
`.podirect` - point out direct to some fix
`.poc` - point out climbing on route
`.pocdirect` - point out climbing direct to some fix
`.pod` - point out descending
`.poddirect` - point out descending direct to some fix

Answers:
`.poa` - point out approved
`.poar` - point out approved reference an aircraft
`.pou` - unable point out
`.rc` - radar contact

**Comment by FriedrichKayak** [comment]
@erikquinn In which context are these used?

**Comment by erikquinn** [comment]
The aliases should send a private message to the specified controller, eg:
`.po mmv curso 15000`
(send MMV sector a private message of)
`Point out, $dist($2) miles $bear($2) of $2, $aircraft, $3 along flight plan route?`
(which yields a message of)
`Point out, 15 miles south of CURSO, N123AD, 15000 along flight plan route?`

Another example:
`.pocdirect 25 rsw FL280 canoa`
(send private message to 25)
`Point out, $dist($2) miles $bear($2) of $2, $aircraft, $calt climbing to $3, direct $4?`
(which yields a message of)
`Point out, 40 miles northeast of RSW, N123AD, 10500 climbing to FL280, direct CANOA?`

In short, gives us the ability to aliasify radar point outs.
